BrainPortal/app/helpers/rich_ui_helper.rb

```diff
@@ -261,6 +261,30 @@ def overlay_content_link(name, options = {}, &block)
     html.html_safe
   end
 
+  # Create a link that will open a dialog box.
+  #
+  #   +link_text+: clickable text that appears on the page in order to open the overlay
+  #   +title+: title that will show up as a header at the top of the overlay
+  #   +description+: text of the overlay content
+  #
+  # Options:
+  #
+  #   +size+: maximum height and width of the overlay
+  def closable_overlay_content_link(link_text, title, description="", options = {size: "50em"})
```

Comment:
This method doesn't allow us to style the link like the normal overlay_content_link() can.

With the roriginal method, I can say

```
overlay_content_link("info", :class => 'action_link') { "description" }
```

and the options are transformed into attributes that are stored in the "A" element. (Try this in the console)

This new helper is hardcoded to only use the size option, and not propagate the rest of the options hash down the call stack.

It should instead extract the size and pass the rest of options down (using dup).

Reply:
Maye better is to configure the size to be an actual argument:

```ruby
def closable_overlay_content_link(link_text, title, description="", size="50em", options={})
```

That way you don't need to adjust options before passing it along to the other helper.